ui/shared/DropdownMenuItems.jsx

```diff
@@ -27,20 +34,42 @@ const DropdownMenuItems = ({
   options,
   pinned,
   namespace,
+  otherPinned,
 }) => (
   <DropdownMenu className="overflow-auto dropdown-menu-height">
     {/* Items pinned to top of dropdown */}
     {pinned.length > 0 && (
       <React.Fragment>
         {pinned.map((item) =>
-          createDropdownItem(item, namespace, updateData, selectedItem),
+          createDropdownItem(
+            item,
+            namespace,
+            updateData,
+            selectedItem,
+            'top-pinned',
+          ),
         )}
         <DropdownItem divider />
       </React.Fragment>
     )}
     {options.map((item) =>
       createDropdownItem(item, namespace, updateData, selectedItem),
     )}
+    {/* Items pinned to bottom of dropdown */}
```

Comment:
@esanuandra I already added comments here in the jsx file but sure will add to test.